example/etcd-restore-operator/restore_cr.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: "etcd.database.coreos.com/v1beta2"
+kind: "EtcdRestore"
+metadata:
+  name: example-etcd-cluster
+spec:
+  clusterSpec:
+    size: 3
+    version: "3.1.8"
+  backupSpec:
+    clusterName: example-etcd-cluster
+    storageType: S3
+    s3:
+      s3Bucket: jenkins-etcd-operator
+      awsSecret: <aws-secret>
```

Comment:
perfer adding `prefix: <s3-prefix>` to indicate prefix support? unless we are change the prefixing strategy.